custom_components/solplanet/sensor.py

```diff
@@ -285,6 +293,7 @@ def create_inverter_entites_description(
             data_field_device_type=INVERTER_IDENTIFIER,
             data_field_data_type="data",
             data_field_path=["flg"],
+            data_field_NaN_value=0xFF,
```

Comment:
shouldn't it be `0xFFFF` ?
```suggestion
            data_field_NaN_value=0xFFFF,
```

Reply:
No, it is 0xFF. You can see in the example output: "flg":255

Reply:
Ok, I've asked because I saw in some modbus documentation (https://api.library.loxone.com/downloader/file/1054/MB001_ASW%20GEN-Modbus-en_V2.1.1.pdf) that it should be 0xFFFF:

![image](https://github.com/user-attachments/assets/cc4306fb-f378-43fb-b227-6845bd91e2f3)
![image](https://github.com/user-attachments/assets/824ae6e0-abce-486c-b275-92bb29d4f18d)

Maybe it is not the latest documentation or our inverters use different standard than described in above file.

Thanks for your contribution :)